packages/dropdowns/src/elements/combobox/utils.ts

```diff
@@ -16,8 +16,7 @@ import { IOptGroupProps, IOptionProps } from '../../types';
  *
  * @returns A string based on `option.value`.
  */
-export const toString = (option: IOption) =>
-  typeof option.value === 'string' ? option.value : JSON.stringify(option.value);
+export const toString = (option: IOption) => option.value;
```

Comment:
Is this utility still used anywhere? It doesn't seem like much of a utility anymore :wink:

Reply:
Won't you want to stroll down memory lane in six months? 😛